tasks/managed/filter-already-released-images/tests/test-filter-no-repositories.yaml

```diff
@@ -0,0 +1,172 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-filter-no-repositories
```

Comment:
Maybe the test name should be `test-filter-keep-on-no-repository` or something. The current name made think it was suppose to check if it was filtered out on repository missing.